inspirehep/dojson/hep/fields/bd5xx.py

```diff
@@ -235,12 +235,19 @@ def license2marc(self, key, value):
 @hep.over('acquisition_source', '^541[10_].')
 def acquisition_source(self, key, value):
     """Immediate Source of Acquisition Note."""
+    def _get_source(value):
+        sources = force_force_list(value.get('a'))
+        sources_without_inspire = [
+            el for el in sources if not el.startswith('inspire')]
+
+        return force_single_element(sources_without_inspire)
+
     return {
-        'source': value.get('a'),
+        'source': _get_source(value),
         'email': value.get('b'),
         'method': value.get('c'),
         'date': value.get('d'),
-        'submission_number': str(value.get('e'))
+        'submission_number': value.get('e'),
```

Comment:
Turns out there was absolutely no reason for this. This used to output`'None'` (a string containing the literal word `None`) when `541` has no `e`. Why? Who knows.